tasks/rh-sign-image/README.md

```diff
@@ -12,6 +12,16 @@ Task to create internalrequests to sign snapshot components
 | requestTimeout  | InternalRequest timeout                                                                   | Yes      | 180                  |
 | concurrentLimit | The maximum number of images to be processed at once                                      | Yes      | 16                    |
 | pipelineRunUid  | The uid of the current pipelineRun. Used as a label value when creating internal requests | No       | -                    |
+| signRegistryAccessPath | The relative path in the workspace to a text file that contains a list of repositories that needs registry.access.redhat.com image references to be signed (i.e. requires_terms=true), one repository string per line, e.g. "rhtas/cosign-rhel9". | No       | -                    |
```

Comment:
vertical alignment please

Reply:
See the other thread - I don't think that's a good idea.

Reply:
Fixed.